### ISSUE TITLE
Vision

### ISSUE THREAD

**Issue by qtrandev** [CodeforSouth/LeafletTransit#47]
Let's discussion the vision for this project

1) Dashboard
Mayor wants to see overall transportation map - filter by transport types, see where all vehicles are.
Civic activist wants to see how bus routes cover the city and how late buses and trains are.

2) User at bus stop
College student taking usual bus to Miami-Dade College wondering why bus hasn't come
New rider following Google Maps direction to take the 150 bus westbound but it didn't arrive

3) Tourist exploring transit options
Visitor from Spain wants to see what they can take to Miami Beach and what they can ride to get around once they get there.
Visitor wants to see all the Points of Interest near where the bus/train drops off.

**Comment by bruncun** [comment]
Been giving this more thought, and I think considering all these cases the best default view would be an overview of all transit systems. We already offer a current location button in similar fashion to google maps, so that should be good enough, right?

**Comment by qtrandev** [comment]
Yes, the user can use the small location button. We can add a bigger "Locate Me" button on the map to help them tap it.

**Comment by dannav** [comment]
I was thinking about this. Why couldn't we take the existing api data and convert it to  GTFS-realtime feed for gmaps to use? We can use our api (which sits on top of the public transit data, add whatever gps data we collect) and convert it to GTFS format.

That's ultimately what we want to accomplish right?

**Comment by qtrandev** [comment]
@dannav Do you want to give it a try? Create a GTFS-realtime feed for City of Miami trolley system which already has realtime data.

I have a basic outline of the GTFS zip schedule data here: https://github.com/qtrandev/OneBusAway/tree/master/GTFS

Realtime data is here:
http://miami.etaspot.net/
You'll need to format that to the GTFS-realtime format which might be hard (https://developers.google.com/transit/gtfs-realtime/?hl=en)
Create this server in your favorite language like Go. The miami-transit-api project already has this server in Ruby (with 1 pull request from Bryce).

Here's a sample of what the feed looks like: https://github.com/qtrandev/OneBusAway/wiki#6-setting-up-a-gtfs-realtime-gtfs-rt-feed (JSON instead of protocol buffers here)

Create an issue and I'll put all the details for you to get started. I'll guide you along if needed.

**Comment by dannav** [comment]
@qtrandev Yep I'm going to take a stab at it tonight
